android/src/main/java/cl/json/ShareFile.java

```diff
@@ -104,12 +107,13 @@ public Uri getURI() {
 
         if(this.isBase64File()) {
             String encodedImg = this.uri.getSchemeSpecificPart().substring(this.uri.getSchemeSpecificPart().indexOf(";base64,") + 8);
+            String filename = this.filename != null ? this.filename : System.nanoTime() + "";
```

Comment:
what happens if user pass a fileName with special characters?

Reply:
@jgcmarins you will have fileName with special characters